docs/basics/gas.md

```diff
@@ -80,6 +85,19 @@ types other than the ones defined by the `evm` module. EVM module transactions c
 as gas is required inherently by the EVM. This check is done by the EVM transactions stateless validation
 (i.e `ValidateBasic`) function as well as on the custom `AnteHandler` defined by Ethermint.
 
+## Gas estimation
+
+Ethereum provides a JSON-RPC endpoint `eth_estimateGas` to help users set up a correct gas limit in their transactions. 
+
+Unfortunately we cannot make use of the SDK `tx simulation` for gas estimation because the pre-check in the Ante Handlers would require a valid signature, and the sender balance to be enough to pay for the gas. But in Ethereum, this endpoint can be called without specifying any sender address.
```

Comment:
```suggestion
Unfortunately, we cannot make use of the SDK `tx simulation` for gas estimation because the pre-check in the Ante Handlers would require a valid signature, and the sender balance to be enough to pay for the gas. But in Ethereum, this endpoint can be called without specifying any sender address.
```